pkg/policy/policy.go

```diff
@@ -76,10 +76,13 @@ func NewClient(cacheDir string, quiet bool, opts ...Option) (*Client, error) {
 	}, nil
 }
 
-func (c *Client) populateOCIArtifact() error {
+func (c *Client) populateOCIArtifact(policyBundleRepository string) error {
```

Comment:
What if adding policyRepo here rather than passing the repo to each method?
https://github.com/aquasecurity/trivy/blob/1ec96d7f5415e2aa3dde70e51915d7837239c163/pkg/policy/policy.go#L49-L54